parser.go

```diff
@@ -285,33 +299,60 @@ func (p *parser) readPythonMultilines(line string, bufferSize int) (string, erro
 	parserBufferPeekResult, _ := p.buf.Peek(bufferSize)
 	peekBuffer := bytes.NewBuffer(parserBufferPeekResult)
 
+	indentSize := 0
 	for {
 		peekData, peekErr := peekBuffer.ReadBytes('\n')
 		if peekErr != nil {
 			if peekErr == io.EOF {
+				p.debug("readPythonMultilines: failed to peek")
+				p.debug("readPythonMultilines: peekData is: '"+string(peekData)+"'")
+				p.debug("readPythonMultilines: value is: '"+line+"'")
```

Comment:
```suggestion
				p.debug(fmt.Sprintf("parser.readPythonMultilines: io.EOF, peekData: %q, line: %q", string(peekData), line))
```

Reply:
I don't think this is necessary, but whatever, I'll make the change to close this out